src/components/Home/index.js

```diff
@@ -1,11 +1,35 @@
 import styles from './Home.module.css'
+import { GitHub, Linkedin } from 'react-feather'
 import Profilepic from '../../assets/Profilepic.jpg'
 
 function Home() {
     return (
-        <div className={styles.container}>
-            <img className={styles.img} src={Profilepic} alt="me" />
-        </div>
+        <section className={styles.homeContainer}>
+            <h1>Sami Fares</h1>
+            <p>
+                I embrace collaboration. My goals are to be better everyday and
+                bridge my passions through code.
+            </p>
+            <section className={styles.imgContainer}>
+                <div>
```

Comment:
The extra Div here has no use. You have an .imgContainer which wraps the img, the div is redundant and you can get the same styling done without it